libindy/src/services/metrics/models.rs

```diff
@@ -12,3 +15,32 @@ impl MetricsValue {
         MetricsValue { value, tags }
     }
 }
+
+#[derive(Serialize, Deserialize, Copy, Clone, Debug)]
+pub struct CommandCounters {
+    pub count: u128,
+    pub duration_ms_sum: u128,
+    pub duration_ms_bucket: [u128; BUCKET_COUNT],
+}
+
+impl CommandCounters {
+    pub fn new(count: u128, duration_ms_sum: u128, duration_ms_bucket: [u128; BUCKET_COUNT]) -> Self {
+        CommandCounters {count, duration_ms_sum, duration_ms_bucket}
```

Comment:
I don't think we need parameters in this method, it should be okay to always initialize with zeros